Team 3749 2023/src/main/java/frc/robot/utils/ColorSensorCommand.Java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.commands;
```

Comment:
why is this named ColorSensorCommand? its a utility class and should be named as such